tagstudio/src/qt/widgets/landing.py

```diff
@@ -87,7 +87,6 @@ def update_logo_color(self, style: str = "mono"):
         Args:
             style (str): = The style of the logo. Either "mono" or "gradient".
         """
-        logo_im: Image.Image = None
```

Comment:
Nit: Why remove this?

Reply:
An Image object will always be assigned to `logo_im` regardless of the condition. it’s unnecessary to assigned `None` to it.